phantazm-zombies/src/main/java/com/github/phantazmnetwork/zombies/equipment/Upgradable.java

```diff
@@ -0,0 +1,7 @@
+package com.github.phantazmnetwork.zombies.equipment;
+
+public interface Upgradable {
```

Comment:
suggestion: expose current level, maximum level, and a setLevel(int) method for more flexibility

Reply:
should setLevel(int) only allow increasing levels?
(also we will likely change from ints to Keys so is this necessary)

Reply:
sure, but then setLevel is kind of misleading - if we're using keys, might want to just have an `upgrade` method instead